README.md

```diff
@@ -156,301 +132,6 @@ Then you can start a hypervisor with:
 $ hypervisor 
 ```
 
-### Apps
```

Comment:
Why have we gotten rid of this section?

Reply:
Also, isn't the `# Transports` and `# App programming API` sections also needed?

Reply:
Added the transport section back in. The API section needs to be fixed afaics.

Reply:
@jdknives yes, these need to be updated. Should we do that in another PR/ticket?